superflore/generators/bitbake/run.py

```diff
@@ -38,16 +39,24 @@
 
 
 def main():
+    os.environ["ROS_OS_OVERRIDE"] = "openembedded"
     overlay = None
     preserve_existing = True
     parser = get_parser('Deploy ROS packages into Yocto Linux')
+    parser.add_argument(
+        '--skip-keys',
```

Comment:
I think `--skip-keys` might be a good candidate for the next patch to pull out of this. Is that possible?

Reply:
Most certainly! Please take a look at https://github.com/ros-infrastructure/superflore/pull/175 .